frontend/src/pages/Products.jsx

```
@@ -1,208 +1,249 @@
+                          src={value.image}
+                          alt={value.name}
+                          className="mx-auto"
+                        />
+                      </CardContent>
+                      <CardHeader>
+                        <CardTitle>{value.name}</CardTitle>
+                        <CardDescription>{value.description}</CardDescription>
+                      </CardHeader>
+                    </div>
+                    <CardFooter>
+                      <Link to={"/product/" + value.id}>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)